dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -39,7 +39,9 @@ export const LiveBlock = ({
 }: Props) => {
 	if (block.elements.length === 0) return null;
 	const palette = decidePalette(format);
-	const blockLink = `${joinUrl(host, pageId)}#block-${block.id}`;
+	const blockLink = `${joinUrl(host, pageId)}?page=with:block-${
+		block.id
+	}#block-${block.id}`;
```

Comment:
Once we start building these URLs, I’m always tempted to use the `URLSearchParams` constructor, which we already polyfill. Maybe it only becomes useful once we have more than one.

Reply:
Yeah, I generally hold off firing the URLSearchParams canon until things get larger. Although in this case prettier is doing some quiiikry things...